pkg/base1/Makefile.am

```diff
@@ -150,13 +155,17 @@ EXTRA_DIST += \
 	pkg/base1/moment.min.js \
 	pkg/base1/react.min.js \
 	pkg/base1/term.min.js \
+	pkg/base1/cockpit-react.min.js \
 	pkg/base1/test-dbus-common.js \
 	$(base_BUNDLE) \
 	$(base_GEN) \
 	$(base_GEN:.js=.min.js) \
 	$(NULL)
 
-update-lib:: update-bower
+REACT_INPUTS = \
+	pkg/base1/cockpit-react-terminal.js
+
+update-lib:: update-bower $(REACT_INPUTS)
```

Comment:
Good point. I can change it to always regenerate `cockpit-react.js` when a component changes.